perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -1310,6 +1333,15 @@ public void deleteGroupFromAutoRegistration(PerunSession sess, Group group) {
 		}
 	}
 
+	@Override
+	public void deleteGroupFromAutoRegistration(PerunSession sess, Group group, ApplicationFormItem formItem) {
+		try {
+			jdbc.update("delete from auto_registration_groups where group_id=? and application_form_item_id=?", group.getId(), formItem.getId());
```

Comment:
A check that the form item exists would be nice (or at least throw a comprehensive exception if not)